packages/strapi-design-system/src/Field/FieldAction.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+
+export const FieldAction = styled.button`
+  border: none;
+  background: transparent;
+  font-size: 1.6rem;
+  margin-left: -100%;
```

Comment:
I hate negative margin 😂  can you explain, why do you need it, please?

Reply:
The Input itself is wrapped in a parent container called "InputWrapper". This input wrapper has "flex" display. We can represent the big picture as following:

![Untitled-2021-03-27-2011](https://user-images.githubusercontent.com/3874873/116560877-16a6f000-a902-11eb-9d2c-63525993b4a4.png)

In order for the Icon to be inside the Input, it needs a way to be moved from the right edge. I know two ways to do that: make the InputWrapper relatively positioned, make the Action absolutely position, and use absolute coordinates or use a negative margin.

I honestly don't have a strong preference here.

Do you have a preferred way? Or even an option that I don't know about? 😊

Reply:
(InputWrapper is now a "row")

Reply:
TBH I prefer the `position relative` (I do it in the input UID), the margin is a bit weird when you inspect the dom (There is no impact in other elements). 

![Screenshot 2021-04-30 at 10 56 48](https://user-images.githubusercontent.com/7756284/116673217-67baf080-a9a3-11eb-95bd-c418935f6748.png)

Reply:
I took another approach and move the border to the parent. Then, the FieldAction only has to "push" things out . It fixes the negative margin issue and the input under the text issue 👍🏻